tests/test_utils/test_data.py

```diff
@@ -1,68 +0,0 @@
-# python3
-# Copyright 2021 InstaDeep Ltd. All rights reserved.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-from dm_env import StepType, TimeStep
-
-from mava.types import OLT
-
-
-def get_expected_parallel_timesteps_1() -> TimeStep:
```

Comment:
I see the code coverage did go up, so I'm guessing we aren't making use of this anymore for some reason?

Reply:
I was about to remove this file we don't need it anymore!

Reply:
Also I found some unecessary mocks I will remove them too

Reply:
Ok sounds good 🙂

Reply:
It was used to test wrapper utils such as  broadcast_timestep_to_all_agents,  convert_seq_timestep_and_actions_to_parallel, and I believe we removed these two tests since they are related to the relation between sequential and parallel env